src/listeners/join-leave-message.ts

```diff
@@ -0,0 +1,76 @@
+import {
+	Colors,
+	EmbedBuilder,
+	GuildMember,
+	PartialGuildMember,
+	userMention,
+} from "discord.js";
+import { Listener } from "../listener.ts";
+import { Config } from "../schemas/config.ts";
+import db from "../db.ts";
+
+const getTargetChannel = async (member: GuildMember | PartialGuildMember) => {
+	const [firstRow] = await db
+		.select({ gatewayChannel: Config.gatewayChannel })
+		.from(Config)
+		.limit(1);
```

Comment:
This query looks like its missing a .where() clause to search for the correct guild id. Also, with the where clause, limit will be unecessary.

It is also a good idea to use [prepared statements](https://orm.drizzle.team/docs/performance) with placeholders for the guild id, as this will help performance quite a bit.

Finally, you can use `.get` to automatically return the first row so you don't need to use destructuring.

Reply:
I can't figure out how to do that properly, I'm doing it like it says in the docs.
It tells me that the `placeholder` method is deprecated (with the message "Use `sql.placeholder` instead.") but the usage of that isn't mentioned in the docs anywhere.

Reply:
As far as I can tell, the API for `sql.placeholder` is exactly the same as `placeholder`. Just `import {sql} from "drizzle-orm"` and use that.